jest.config.js

```diff
@@ -0,0 +1,192 @@
+/*
```

Comment:
This is the standard jest config file, with a slight change to line 150, where the file matching is done.